build/teamcity-roachtest-invoke.sh

```diff
@@ -1,23 +1,14 @@
 #!/usr/bin/env bash
-set -euo pipefail
+set -euxo pipefail
```

Comment:
Do you want to keep the `-x` here? I suppose it doesn't hurt.

Reply:
No strong opinion, mostly added it because the calling scripts used `-x` so I figured they'd want to see the final invocation. Remove it now.